app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -1281,28 +1281,33 @@ public void onClick(DialogInterface dialogInterface, int i) {
                                 alertDialog.show();
                                 AlertDialogsHelper.setButtonTextColor(new int[]{DialogInterface.BUTTON_POSITIVE, DialogInterface.BUTTON_NEGATIVE, DialogInterface.BUTTON_NEUTRAL}, getAccentColor(), alertDialog);
                             } else {
-                                int index = file.getPath().lastIndexOf("/");
-                                String path = file.getPath().substring(0, index);
```

Comment:
why were these lines removed only to be added later?

Reply:
I used an additional if condition. So all the lines have been shifted down. It's just appearing to be this way here. I have just added lines, not deleted any.

Reply:
cool.